lib/writer.js

```diff
@@ -67,7 +67,7 @@ exports.markdown = function (version, commits, options) {
     Object.keys(this.types[type]).forEach(function (category) {
       var prefix = '*';
       var nested = types[type][category].length > 1;
-      var categoryHeading = '* **' + category + ':**';
+      var categoryHeading = prefix + (Boolean(category) ? ' **' + category + ':**' : '');
```

Comment:
Well... your lint setup told me to use `Boolean(category)` instead of `!!category`.

But you're right the empty `category` string is falsy enough ;)